.github/workflows/ci.yml

```diff
@@ -157,9 +159,9 @@ jobs:
     steps:
       - name: checkout
         uses: actions/checkout@v2
-      - name: hare+mining system tests
-        run: make dockertest-hare-mining
```

Comment:
Was this test removed? Or replaced?

Reply:
we don't run tests together in pairs anymore